galaxy_ng/tests/unit/api/test_api_ui_settings.py

```diff
@@ -0,0 +1,20 @@
+from django.conf import settings
+
+from .base import BaseTestCase, get_current_ui_url
+
+
+class TestUiFeatureFlagsView(BaseTestCase):
+    def setUp(self):
+        super().setUp()
+        self.original_setting = settings.GALAXY_FEATURE_FLAGS
+        settings.GALAXY_ENABLE_UNAUTHENTICATED_COLLECTION_ACCESS = True
```

Comment:
wasn't supposed to be like this?
```suggestion
        self.original_setting = settings.GALAXY_ENABLE_UNAUTHENTICATED_COLLECTION_ACCESS
        settings.GALAXY_ENABLE_UNAUTHENTICATED_COLLECTION_ACCESS = True
```

Reply:
Ha, yeah.  :smile: